packages/appstore/src/components/appstore-banner/appstore-banner.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import SVGMigrationBanner from './svg-migration-banner';
+
```

Comment:
are we adding some test cases for this? 🧪

Reply:
will add the test case once the API is ready. Now we can't really test it since it's just a dummy data 😓